contrib/lowdown/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "lowdown"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "PREFIX=/usr",
+    "MANDIR=/usr/share/man",
+]
```

Comment:
you have to put LDFLAGS= here too because those aren't taken from env. you can do that in init_configure and using self.get_ldflags(shell=True)

Reply:
ok.